### PR TITLE
refactor: outputData as a proper PWMetrics method

### PR DESCRIPTION
**Pull request by GuillaumeAmat**
Related to: https://github.com/paulirish/pwmetrics/pull/194#discussion_r224896044

### CODE REVIEW

File: lib/utils/fs.ts [hunk 1]
```diff
@@ -17,7 +23,20 @@ function getConfigFromFile(fileName: string = 'package.json') {
...
+
 }
 
-module.exports = { getConfigFromFile };
+export function writeToDisk(fileName: string, data: string) {
+  return new Promise((resolve, reject) => {
+    const filePath = path.join(process.cwd(), fileName);
+
+    try {
+      fs.writeFileSync(filePath, data);
```
Comment (denar90) [inline lib/utils/fs.ts]: wdyt if we wrap `fs.writeFile` with `micro-promisify` (we already have it as dep) and stick to async/await here?

File: lib/utils/fs.ts [hunk 2]
```diff
@@ -17,7 +23,20 @@ function getConfigFromFile(fileName: string = 'package.json') {
       return config.pwmetrics || {};
     else return config;
   } else throw new Error(`Invalid config from ${fileName}`);
-  
+
 }
 
-module.exports = { getConfigFromFile };
+export function writeToDisk(fileName: string, data: string) {
```
Comment (denar90) [inline lib/utils/fs.ts]: 😍 to see it here :)

Comment (GuillaumeAmat) [reply lib/utils/fs.ts]: 😊

File: lib/utils/fs.ts (hunk 1 above)
Comment (GuillaumeAmat) [reply lib/utils/fs.ts]: Sure, I just pushed it :)

File: lib/utils/fs.ts [hunk 3]
```diff
@@ -17,7 +25,20 @@ function getConfigFromFile(fileName: string = 'package.json') {
...
+  return new Promise(async (resolve, reject) => {
```
Comment (denar90) [inline lib/utils/fs.ts]: I meant do we need Promise here at all?
We can write inside function just
```
try { 
  const filePath = path.join(process.cwd(), fileName);
  await promisify(fs.writeFile)(filePath, data);
  logger.log(getMessageWithPrefix('SUCCESS', 'SAVED_TO_JSON', filePath));
} catch (err) {
      throw (err);
    }
```

Comment (GuillaumeAmat) [reply lib/utils/fs.ts]: Oh, ok it makes sense you're right. The promise is removed there but the whole thing has to be awaited back to the call of `outputData` line 108 of `lib/index.ts` though.

**Pull request merged**